components/widgets/land-cover/tree-cover-density/index.js

```diff
@@ -0,0 +1,67 @@
+import { getTreeCoverDensity } from 'services/analysis-cached';
+
+import {
+  POLITICAL_BOUNDARIES_DATASET,
+  TROPICAL_TREE_COVER,
+} from 'data/datasets';
+import {
+  DISPUTED_POLITICAL_BOUNDARIES,
+  POLITICAL_BOUNDARIES,
+  TROPICAL_TREE_COVER_METERS,
+  TROPICAL_TREE_COVER_HECTARE,
+} from 'data/layers';
+
+import getWidgetProps from './selectors';
+
+export default {
+  widget: 'treeCoverDensity',
+  title: 'Tree cover density in {location}',
+  categories: ['land-cover'],
+  types: ['country', 'wdpa', 'aoi'],
+  admins: ['adm0', 'adm1', 'adm2'],
+  large: true,
+  visible: ['dashboard', 'analysis'],
+  chartType: 'composedChart',
+  colors: 'density',
+  settingsConfig: [
+    {
+      key: 'landCategory',
+      label: 'Land Category',
+      type: 'select',
+      placeholder: 'All categories',
+      clearable: true,
+      border: true,
+    },
+  ],
+  pendingKeys: ['threshold', 'years'],
+  refetchKeys: ['landCategory'],
+  dataType: 'gain',
```

Comment:
This may not do anything, but you could change the `dataType` to `density` because it would be more "true". Just in case, in the future, there's some setting related to gain that makes something appear different.